test/ussd_higherhealth_healthcheck.test.js

```diff
@@ -389,14 +392,227 @@ describe("ussd_higherhealth_healthcheck app", function () {
                 })
                 .run();
         });
-        it("should go to state_fever", function () {
+        it("should go to state_university", function () {
             return tester
                 .setup.user.state("state_age")
+                .setup.user.answers({
+                    state_province: 'ZA-GT',
+                })
+                .input("1")
+                .check.user.state("state_university")
+                .run();
+        });
+    });
+    describe("state_university", function () {
+        it("should ask for a users university", function () {
+            return tester
+                .setup.user.answers({
+                    state_province: 'ZA-GT',
+                })
+                .setup.user.state("state_university")
+                .check.interaction({
+                    state: "state_university",
+                    reply: [
+                        "Select your university.",
+                        "",
+                        "Reply:",
+                        "1. Central Johannesburg",
+                        "2. Ekurhuleni East",
+                        "3. Ekurhuleni West",
+                        "4. North West University (NWU)",
+                        "5. Sedibeng",
+                        "6. More"
+                    ].join("\n"),
+                    char_limit: 160
+                })
+                .check.user.answer('state_province', 'ZA-GT')
+                .run();
+        });
+        it("should display an error on invalid input", function () {
+            return tester
+                .setup.user.state("state_university")
+                .setup.user.answers({
+                    state_province: 'ZA-GT'
+                })
+                .input("A")
+                .check.interaction({
+                    state: "state_university",
+                    reply: [
+                        "Select your university.",
+                        "",
+                        "Reply:",
```

Comment:
I see for invalid input, we're using the same text as we do for the initial question. Is that intended?

Reply:
yes